### DISCUSSION TITLE
Should we ditch French ?

### DISCUSSION THREAD

**Discussion by ZibanPirate** [dzcode-io/dzcode.io#423] [Q&A] [answered]
I don't see a reason for adding French to the website, most of us speak English or Arabic very well, adding another language will only mean more things to maintain, and unless you really need it, I don't think we will have time to support French (articles/docs translations ...etc).

@dzcode-io/maintainers what do you think?

**Comment by souhaib-benbouzid** [comment, marked as answer]
yeap, go for it. most of the french projects use English as the main language so it's just a hustle to add french with no real added value.

**Reply by ZibanPirate** [reply]
I guess, the maintainers agreed on removing it.

We can later add it if we feel the need to do so.
